src/main/java/org/tikv/common/region/RegionStoreClient.java

```diff
@@ -82,7 +86,8 @@ public ByteString get(BackOffer backOffer, ByteString key, long version) {
       GetResponse resp = callWithRetry(backOffer, TikvGrpc.METHOD_KV_GET, factory, handler);
 
       if (resp == null) {
-        this.regionManager.onRequestFail(context.getRegionId(), context.getPeer().getStoreId());
+        this.regionManager.onRequestFail(
+            region.getContext().getRegionId(), region.getContext().getPeer().getStoreId());
```

Comment:
The call looks ugly especially consider getcontext is actually a build action. Make context a local var.